platform/darwin/src/MGLTileSource.h

```diff
@@ -141,6 +142,37 @@ typedef NS_ENUM(NSUInteger, MGLTileCoordinateSystem) {
     MGLTileCoordinateSystemTMS
 };
 
+
+/**
+ An `NSNumber` object containing an unsigned integer that specifies the encoding
+ formula for raster-dem tilesets. The integer corresponds to one of
+ the constants described in `MGLDEMEncoding`.
+
+ The default value for this option is `MGLDEMEncodingMapbox`.
+
+ This option is not supported by the TileJSON spec.
+ */
+extern MGL_EXPORT const MGLTileSourceOption MGLTileSourceOptionDEMEncoding;
+
+/**
+ The encoding formula used to generate the raster-dem tileset
+*/
+
+typedef NS_ENUM(NSUInteger, MGLDEMEncoding) {
+
+    /**
+     Raster tiles generated with the Mapbox encoding formula where
+     elevaion = -10000 + ((R * 256 * 256 + G * 256 + B) * 0.1)
+    */
+    MGLDEMEncodingMapbox = 0,
+
+    /**
+     Raster tiles generated with the Mapzen Terrarium encoding formula where
```

Comment:
Should we link to [this documentation](https://mapzen.com/documentation/terrain-tiles/formats/#terrarium) or [this GitHub repository](https://github.com/mapzen/terrarium/)?

Reply:
in GL JS I linked to https://aws.amazon.com/public-datasets/terrain/ due to the mapzen shutdown 🤷‍♀️

Reply:
Sure, that works too.